src/containers/App/App.js

```diff
@@ -46,6 +47,12 @@ const App = (props) => {
         `wss://${process.env.REACT_APP_P2P_RIPPLED_HOST}:${process.env.REACT_APP_RIPPLED_WS_PORT}`,
       ])
     : undefined
+  socket.clioSocket =
```

Comment:
I'd like to limit the number of new env vars we have. Instead of this, could we just make sure the p2p URL is a Clio node? I assume we want all versions of the explorer to use Clio eventually.

Reply:
Does clio have the gateway call?

Reply:
Yes

Reply:
@mvadari i have tried to use the cliosocket. However it doesnt work with gateway_balances. Also currently this cliosocket could only be connected through ws, not wss. Should we just it leave off for now?